CHANGELOG.md

```diff
@@ -1,4 +1,22 @@
-<a name="3.0.0"></a>
+<a name="3.0.1"></a>
+# 3.0.1
+
+## OIDCStrategy
+
+### New features
+
+* specify tenant per request 
+
+  Now you can specify the tenant per request, using the `tenantIdOrName` option in `passport.authenticate`. More details on the usage can be found in README.md. This brings us two additional benefits:
```

Comment:
Instead of "This brings us two additional benefits:", "tenantIdOrName enables two features:"